openmetadata-ui/src/main/resources/ui/src/components/common/ResizablePanels/ResizablePanels.tsx

```diff
@@ -111,7 +111,7 @@ const ResizablePanels: React.FC<ResizablePanelsProps> = ({
           }}>
           {!hideSecondPanel && (
             <PanelContainer overlay={secondPanel.overlay}>
-              {secondPanel.children}
+              <div className="p-l-sm">{secondPanel.children}</div>
```

Comment:
This will have effects everywhere

Reply:
we need that every to avoid the dragger overriding the content 
<img width="483" alt="image" src="https://github.com/user-attachments/assets/126a8819-6ba3-43be-b995-1cf87ec6e16c">